apps/vetrina/src/Vetrina/Main.purs

```diff
@@ -193,7 +193,7 @@ pollOrder setState state@{ logger } (Right order) = do
         chooseAccountStatus user
           | user.hasCompletedRegistration = ExistingAccount
           | otherwise = NewAccount
-    OrderFailed    -> liftEffect do
+    OrderFailed reason -> liftEffect do
```

Comment:
Now we need to handle the `SubscriptionExistsError` case

Reply:
How could we do it?

In #299 you said we should just close `vetrina` if I'm correct, but doesn't this lack of transparency to the user? If she made an order, maybe she would like to know what happend to it